packages/webpack/test/e2e-hack.spec.js

```diff
@@ -0,0 +1,59 @@
+const test = require('ava')
+const { scaffold, runScriptWithSES, runScript } = require('./scaffold.js')
+const webpackConfigDefault = require('./fixtures/main/webpack.config.js')
+
+test.before(async (t) => {
+  const webpackConfig = {
+    ...webpackConfigDefault,
+    entry: {
+      hack1: './hack1.js',
+      hack2: './hack2.js',
+      hack3: './hack3.js',
+    },
+  }
+  await t.notThrowsAsync(async () => {
```

Comment:
I mean, if it failed, the hook would fail.  unsure if wrapping in an assertion is more helpful or harmful

Reply:
The difference is, without this you get "Rejected promise returned by test" in the result listing which is significantly less helpful